core/src/main/scala/org/apache/spark/rdd/CoalescedRDDPublic.scala

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Yea actually many of the naming in spark core is wrong but we never bothered changing them. Usually SomeWord.scala means there is a class named SomeWord. The scala style guide actually recommends when there are multiple classes that are part of a coherent group, start with lowercase (similar to a lot of c++ naming guides).